api/FileSystemDirectoryReader.json

```diff
@@ -57,10 +57,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/FileSystemDirectoryReader/readEntries",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": false
```

Comment:
Looking at the data, it seems that `FileSystemDirectoryReader` was defined as having a `webkit` prefix in Chromium, however it seems that the correct data would actually be an alternative name of `DirectoryReader`.  Agreed?

Reply:
This case is like those discussed in https://github.com/mdn/browser-compat-data/issues/3528. The name of the interface doesn't matter much because of `[NoInterfaceObject]`, although you *can* see if you stringify an instance. That the interface object isn't exposed does matter a little bit because it makes feature detection harder. So, I guess I'd say the `readEntries` method is simply supported, and the `FileSystemDirectoryReader` interface should also be marked as supported but with a note that the interface object isn't exposed.

Note that the prefix bit is several layers away from this interface:
https://wicg.github.io/entries-api/#html-forms

Reply:
It would be good to make those non-mechanical changes in separate PRs, so that this PR contains only a subset of the changes created by the script you linked.